ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/aaa-authorization.j2

```diff
@@ -13,4 +13,10 @@ no aaa authorization config-commands
 {%     if aaa_authorization.commands.all_default is arista.avd.defined %}
 aaa authorization commands all default {{ aaa_authorization.commands.all_default }}
 {%     endif %}
+{%     if aaa_authorization.commands.privilege is arista.avd.defined %}
+{%         for command_level in aaa_authorization.commands.privilege %}
```

Comment:
When ever there is a for loop to be used in eos_cli_config_gen templates, we use [natural_sort](https://avd.sh/en/releases-v3.x.x/plugins/index.html#natural_sort-filter) filter.

This does a arista.avd.defined test. So you can modify your code(both lines 16,17) to 
``` for command_level in aaa_authorization.commands.privilege | arista.avd.natural_sort ```
This sorting filter is required to make sure that the commands are sorted before the config files are generated.